manifests/init.pp

```diff
@@ -6,7 +6,7 @@
 #
 #   include vmware_fusion
 class vmware_fusion (
-  $version = '6.0.2-1398658',
+  $version = '6.0.3-1747349'
 ) {
   package { 'VMware Fusion':
     source   => "https://s3.amazonaws.com/boxen-downloads/vmware/VMware-Fusion-${version}-light.dmg",
```

Comment:
are you able to update this URL to not link to the `boxen-downloads` S3 bucket and instead use the vmware URL?